snapshots/oecd/2025-03-26/govt_glance_public_finance.csv.dvc

```diff
@@ -0,0 +1,30 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: "Public finance main indicators - Government at a glance, Yearly updates"
+    description: |-
+      This dataset represents a selection of indicators as presented in the 2023 edition of the Government at a Glance publication related to the public finance under the System of national accounts (SNA). Government at a Glance provides a dashboard of indicators assembled to help decision makers and the public analyse and benchmark governments across OECD members and partner countries. This eighth edition provides internationally comparative data on a range of public governance, tools and practices to help identify both strengths and weaknesses in democratic governance. The Structure and indicators section of the Government at a Glance publication, describes its framework and the structure by chapters, as well as the changes compared to the previous 2021 edition.The 2023 edition presents a new structure around these three broad categories: 1) Trust and satisfaction with public services (trust in public institutions, satisfaction with public services); 2) Achieving results with good governance practices (governance of the policy cycle, regulatory governance, budgeting practices, managing public procurement, infrastructure planning and delivery, digital government and open government data); and 3) What resources public institutions use and how are they managed (public finance, public employment, managing human resources). Further information on Government at a Glance 2023 edition via a dedicated web platform, which includes a Government at a Glance Data dashboard (for a selection of indicators in an interactive format) as well as Country fact sheets (key data by country), can be found at: [Government at a Glance website](https://www.oecd.org/en/publications/2023/06/government-at-a-glance-2023_da193b0d.html).
```

Comment:
A few potential typos (that apply also to the other snapshots, I suppose):
* System of national accounts -> System of National Accounts (?)
* Missing space ".The 2023"
* The last sentence "Further information..." seems to be missing a verb, e.g. "...can be found..."

You could also add more line breaks (especially the list of categories).

Reply:
Thanks. It comes verbatim, but I corrected some og these points.